### ISSUE TITLE
Create geomap component for mentors

### ISSUE THREAD

**Issue by 10-seconds** [10secondsofcode/coding-ai#18]
Create the geomap components based on mentors location and it will help to visualize the mentors location with count.

**Comment by dishankmehta** [comment]
Hey I am interested in doing this. Can you elaborate more, also do we the data available to show on the map?

**Comment by 10-seconds** [comment]
yes, we have data in response/reposne.js file, So based on mentors country, we have to created the geomap.

**Comment by dishankmehta** [comment]
Okay. I will do this any preference on which library to use for geomap?

**Comment by dishankmehta** [comment]
Okay and do you want a chart or a map on which pins should be shown with counts?

**Comment by 10-seconds** [comment]
@dishankmehta we want map when hover will displaying the count.

**Comment by 10-seconds** [comment]
> Okay. I will do this any preference on which library to use for geomap?

You will use some standard lib your wish, but make sense.
